src/modules/add-update-delete.js

```diff
@@ -0,0 +1,16 @@
+export const addToList = (list) => {
+  const input = document.querySelector('#todo');
+  const { value } = input;
```

Comment:
- You could have declared this in one line, using destructing, you didn't have to first save it input